src/main/api/2/studio-api-2.yaml

```diff
@@ -3860,6 +3860,86 @@ paths:
         '500':
           $ref: '#/components/responses/InternalServerError'
 
+  /content/children_by_paths:
+    post:
+      tags:
+        - content
+      summary: Get list of children for given item paths
+      description: 'Required permission "get_children" and "site member"'
+      operationId: getChildrenByPaths
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                siteId:
+                  description: Site ID
+                  type: string
+                localeCode:
+                  description: |-
+                    filter children by locale code, or show source items (not translation) if translation to
+                    specified locale code does not exist
+                  type: string
```

Comment:
If we are going to refresh multiple widgets at once, we could require it per path. Per design, each widget has its own locale selector. It is not set globally but per widget.

Originally, the localeCode is part of the individual requests. Is there a performance reason to exact it from the individual request? Why change it?